### ISSUE TITLE
Fragment 1.3.4 causes a crash in Instabug

### ISSUE THREAD

**Issue by carlonzo** [Instabug/Instabug-Android#379]
This issue is more an FYI, as I believe the issue is not caused by your library. wondering if you can find a workaround until a proper fix comes from the official fragment library.

### Steps to Reproduce the Problem

update `androidx.fragment:fragment` library to `1.3.4`
shake and report a bug on Instabug

you should see a crash with this stacktrace:
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'boolean java.lang.String.equals(java.lang.Object)' on a null object reference
       at androidx.fragment.app.FragmentTransitionImpl.setNameOverridesReordered(FragmentTransitionImpl.java:192)
       at androidx.fragment.app.DefaultSpecialEffectsController.startTransitions(DefaultSpecialEffectsController.java:665)
       at androidx.fragment.app.DefaultSpecialEffectsController.executeOperations(DefaultSpecialEffectsController.java:114)
       at androidx.fragment.app.SpecialEffectsController.executePendingOperations(SpecialEffectsController.java:294)
       at androidx.fragment.app.FragmentManager.executeOpsTogether(FragmentManager.java:2202)
       at androidx.fragment.app.FragmentManager.removeRedundantOperationsAndExecute(FragmentManager.java:2100)
       at androidx.fragment.app.FragmentManager.execPendingActions(FragmentManager.java:2002)
       at androidx.fragment.app.FragmentManager$5.run(FragmentManager.java:524)
       at android.os.Handler.handleCallback(Handler.java:938)
       at android.os.Handler.dispatchMessage(Handler.java:99)
       at android.os.Looper.loop(Looper.java:233)
       at android.app.ActivityThread.main(ActivityThread.java:8010)
       at java.lang.reflect.Method.invoke(Method.java)
       at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:631)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:978)
```

I've also reported this here https://issuetracker.google.com/issues/188719925

### SDK Version
10.5.0
### Android Version
11
### Device Model
OnePlus

**Comment by MShannan** [comment]
Hello @carlonzo 
Thanks for reporting this,
We created a SNAPSHOT that have a workaround for this issue until a proper fix comes from the official fragment library.
 implmentation 'com.instabug.library:instabug:10.5.0.3231-SNAPSHOT'
you will need to add these lines to your app build.gradle before your dependencies
repositories {
    ....
    maven {
        url "https://oss.sonatype.org/content/repositories/snapshots"
    }
}
This fix will be included in our next official release that should come out in the next few days.
I will close this issue for now, please feel free to reopen it again if you have any further issues.
Thank You 😃

**Comment by mzelzoghbi** [comment]
Hello @carlonzo 
We just shipped 10.5.1 that contains the fix for this issue.

Thanks

**Comment by carlonzo** [comment]
Thanks!
